ntp-pools.sh

```diff
@@ -18,6 +18,6 @@
 # Red Hat Author(s): Chris Lumens <clumens@redhat.com>
 
 # Fails in rhel8 with "Unknown command: timesource"
-TESTTYPE="time coverage fedora-only"
+TESTTYPE="time coverage fedora-only rhel-9-only"
```

Comment:
fedora-only results in skipping of the test on rhel 9, see also https://github.com/rhinstaller/kickstart-tests/pull/541#issuecomment-882551874